src/main/java/picard/illumina/parser/IlluminaDataProviderFactory.java

```diff
@@ -117,7 +118,7 @@
     private final IlluminaFileUtil fileUtil;
 
 
-    private List<Integer> availableTiles;
+    private final List<Integer> availableTiles;
```

Comment:
I wonder everywhere we have List<Integer> for tiles if we can/should just replace to `int[]`?